modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp

```diff
@@ -537,6 +537,24 @@ renderResponse.setTitle(headerTitle);
 						/>
 					</aui:fieldset>
 
+					<c:if test="<%= FFExpirationDateReviewDateConfigurationUtil.expirationDateEnabled() || FFExpirationDateReviewDateConfigurationUtil.reviewDateEnabled() %>">
+						<aui:fieldset collapsed="<%= true %>" collapsible="<%= true %>" label="expiration-date">
+							<c:if test="<%= FFExpirationDateReviewDateConfigurationUtil.expirationDateEnabled() %>">
+								<liferay-ui:error exception="<%= DLExpirationDateException.class %>" message="please-enter-a-valid-expiration-date" />
+
+								<p class="mb-5 text-secondary">
+									<liferay-ui:message key="including-an-expiration-date-will-allow-your-documents-or-media-to-expire-automatically-and-become-unpublished" />
+								</p>
+
+								<aui:input dateTogglerCheckboxLabel="never-expire" disabled="<%= dlEditFileEntryDisplayContext.isNeverExpire() %>" name="expirationDate" wrapperCssClass="expiration-date mb-3" />
```

Comment:
We don't use `expiration-date` for anything maybe something is missing in CSS like in journal

https://github.com/boton/liferay-portal/blob/master/modules/apps/journal/journal-web/src/main/resources/META-INF/resources/css/main.scss#L265-L270
```scss
.expiration-date,
.review-date {
	.form-group-autofit {
		margin-bottom: 8px;
	}
}
```

Reply:
it's better to add the css to D_L_web or delete the css on the tag?